python/test/t_CovarianceModel_std.expout

```diff
@@ -115,6 +115,26 @@ parameterDescription= [scale_0,scale_1,amplitude_0]
 myModel( [-1,-2] ,  [3,5] )= class=CovarianceMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[0.0471566]
 dCov = class=Matrix implementation=class=MatrixImplementation name=Unnamed rows=2 columns=1 values=[0.0490014,0.0857524]
 dCov (FD)= class=Point name=Unnamed dimension=2 values=[0.0490015,0.0857528]
+dCov/dP= [[ 0.098   ]
+ [ 0.3001  ]
+ [ 0.01572 ]]
+myDefautModel =  FractionalBrownianMotionModel(scale=[2], amplitude=[3], exponent=[0.25])
+myModel =  FractionalBrownianMotionModel(scale=[2], amplitude=[3], exponent=[0.25])
+active= [0,1]
+parameter= [2,3]
```

Comment:
Should FractionalBrownianMotionModel::getParameter returns only scale + amplitude ?

Reply:
done